Lesson-7/lesson-7-new/index.html

```diff
@@ -0,0 +1,101 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+    <link rel="stylesheet" href="style.css">
+    <article class="top">
+    <div class="container_heading">
+        <img id="logo_img" class="logo" src="LinkedSprite.svg#logo_img" alt="logo">
+        <p class="stays"><a href=""> Stays</a> </p>
+        <p class="attractions"> <a href="">Attractions</a> </p>
+        <img class="night_img" src="LinkedSprite.svg#moon" alt="night">
+        <img class="account" src="LinkedSprite.svg#account" alt="accountcircle">
+    </div>
+    <div class="container_middle">
+        <p class="heading">Discover stays to live, work or just relax</p>
+        <div class="label">
+            <label id="search" for="search" class="search">Your destination or hotel name</label>
+            <label id="date" for="date">Check-in — Check-out</label>
+        </div>
+        <div class="place_to_request">
+            <input type="search" name="search_bar" placeholder="New York">
+            <input class="calendar" type="date" name="calendar" placeholder="Check-in — Check-out">
+            <input type="text" name="text" placeholder="Adults — Children — Room">    
+            <input class="button" type="submit" value="Search" placeholder="Search">
+        </div>
+        <div class="phone_app">
+        <img class="icon_google" src="image/google-play-badge.png" alt="google">
+        <img class="icon_app_store" src="image/Download_on_the_App_Store_Badge_US-UK_RGB_blk_092917 1.png" alt="app">
+        </div>
```

Comment:
don't hide it for small and middle displays

Reply:
and align horizontally
![image](https://user-images.githubusercontent.com/20189313/128646205-5c4fc04d-9bb7-4d7d-bc99-b7c642531422.png)

Reply:
the same for small displays.
![image](https://user-images.githubusercontent.com/20189313/128646220-f74d90c1-763f-45ac-ac33-4d3a010394f6.png)

Reply:
you shouldn't have horizontal scrolling anywhere